fapi-client/src/test/scala/com/gu/facia/api/utils/FaciaContentUtilsTest.scala

```diff
@@ -0,0 +1,357 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.{Tag, Content}
+import com.gu.facia.api.models.{SupportingCuratedContent, CuratedContent, LatestSnap, LinkSnap}
+import org.joda.time.DateTime
+import org.scalatest.{FreeSpec, Matchers}
+
+class FaciaContentUtilsTest extends FreeSpec with Matchers {
+
+  def makeLinkSnap(linkSnapId: String) = LinkSnap(
```

Comment:
This is a *lot* of setup. Worth thinking about whether this can be pulled out into fixtures for the tests that could then be shared?